source/fundamentals/crud/write-operations/embedded-arrays.txt

```diff
@@ -84,15 +84,15 @@ The First Matching Array Element
 
 To update the first array element that matches your query filter, use the
 positional ``$`` operator. The array field must appear as part of your
-query document to use the positional ``$`` operator. 
+query filter to use the positional ``$`` operator. 
 
 Example
 ```````
 
 The following example performs these actions:
 
 - Query for a document with a ``qty`` field containing the value "18"
-- Decrement the first array value that matches the query document by "3"  using the ``inc()`` method of the ``Updates`` builder and the positional ``$`` operator 
+- Decrement the first array value that matches the document specified by the query filter by "3"
```

Comment:
I think "matches the document" isn't correct here if you're referring to the document that the query filter matches. The document isn't specified by the query filter, it's matched by it.
E.g.
"Decrement the first array value in the document matched by the query filter by "3""